Cataclysm/APLs/ArmsWarrior.simc

```diff
@@ -0,0 +1,49 @@
+Smufrik Arms warrior
+
+actions.precombat+=/berserker_stance
+actions.precombat+=/commanding_shout,if=assigned_shout.commanding_shout&buff.my_battle_shout.down&buff.my_commanding_shout.down
+actions.precombat+=/battle_shout,if=assigned_shout.battle_shout&buff.my_battle_shout.down&buff.my_commanding_shout.down
+
+actions+=/run_action_list,name=aoe,if=active_enemies>2
+actions+=/call_action_list,name=init
+actions+=/sunder_armor,if=variable.should_sunder&variable.emergency_sunder
+actions+=/pummel
+actions+=/run_action_list,name=berserker_stance,if=buff.berserker_stance.up
+actions+=/run_action_list,name=battle_stance,if=buff.battle_stance.up
+
+actions.init+=/variable,name=time_to_die,value=(debuff.training_dummy.up&300)|target.time_to_die
+actions.init+=/variable,name=execute_phase,value=target.health.pct<20
+actions.init+=/variable,name=should_sunder,value=settings.debuff_sunder_enabled&variable.time_to_die>((5-debuff.sunder_armor.stack)*(1.5+latency))+3
+actions.init+=/variable,name=build_sunder,value=!debuff.major_armor_reduction.up|(debuff.sunder_armor.up&debuff.sunder_armor.stack<5)
+actions.init+=/variable,name=maintain_sunder,value=!variable.build_sunder&debuff.sunder_armor.stack=5&debuff.sunder_armor.remains<7&cooldown.bloodthirst.remains&cooldown.whirlwind.remains&!buff.bloodsurge.up
+actions.init+=/variable,name=emergency_sunder,value=debuff.sunder_armor.up&debuff.sunder_armor.remains<1.5

```

Comment:
I generally prefer not to do these separate action lists for variables because they're actually computationally expensive due to the way action list forking works in the addon.  These could just be dropped into the same place in the default (`actions+=/`) list.

Reply:
Ya, I have noticed that, going to move the back down to the actions lists